fixtures/yamlFileForCustomMandatoryFieldTest/jira.yaml

```diff
@@ -5,14 +5,17 @@ snyk:
     maturityFilter: proof-of-concept
     type: vuln #what snyk issue types should we consider to ticket
     priorityScoreThreshold: 20
-    remoteUrl: github.com/owner/repo # URL to the remote repository (when not detected)
 jira:
     jiraTicketType: Task
-    jiraProjectID: 15698
-    assigneeId: 1238769
+    jiraProjectID: "15698"
```

Comment:
why did the format change? The docs now don't match https://github.com/snyk-tech-services/jira-tickets-for-new-vulns#jirayaml

Reply:
I will update the doc

Reply:
did the format actually change and existing scans will fail? Or the documentation was wrong? 
Trying to understand if this is a breaking change to all existing config files.